.changeset/stale-wasps-hammer.md

```diff
@@ -0,0 +1,6 @@
+---
+"@cloudoperators/juno-app-heureka": minor
+"@cloudoperators/juno-app-greenhouse": patch
+---
+
+Optimized GraphQL queries by separating the retrieval of list data from the details panel information. This update sends two requests with distinct queries, improving overall performance. Additionally, made minor adjustments to naming conventions and informational fields.
```

Comment:
Please remember that the text you write here gets added in the changelog of Heureka. It should have an external focus, meaning the text should describe the effect the change has for users of the app. It also shouldn't bee too verbose if not absolutely necessary. All the extra internal information that might be interesting to developers of Heureka should go into the PR and/or issue. My suggestion would be:
```suggestion
Improved overall performance of Services and IssueMatches views by optimizing the underlying GraphQL queries
```